### ISSUE TITLE
Validation error on hello_triangle sample

### ISSUE THREAD

**Issue by rncar** [KhronosGroup/Vulkan-Samples#599]
Resizing this sample emits a validation error:

```
$ ./build/linux/app/bin/Release/x86_64/vulkan_samples sample hello_triangle

[info] Logger initialized
[info] Initializing vulkan instance.
[info] Enabled Validation Layers:
[info] Found GPU: Intel(R) UHD Graphics 770 (ADL-S GT1)
[info] Initializing vulkan device.

VUID-vkAcquireNextImageKHR-semaphore-01286(ERROR / SPEC): msgNum: -370888023 - Validation Error: [ VUID-vkAcquireNextImageKHR-semaphore-01286 ] Object 0: handle = 0xb991fa0000000024, type = VK_OBJECT_TYPE_SEMAPHORE; | MessageID = 0xe9e4b2a9 | vkAcquireNextImageKHR: Semaphore must not be currently signaled. The Vulkan spec states: If semaphore is not VK_NULL_HANDLE it must be unsignaled (https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-vkAcquireNextImageKHR-semaphore-01286)
    Objects: 1
        [0] 0xb991fa0000000024, type: 5, name: NULL
```

**Comment by SaschaWillems** [comment]
Thanks for raising this issue. Do you get this on every resize, or only when e.g. minimizing or maximizing the window?

**Comment by rncar** [comment]
resizing.

**Comment by mokafolio** [comment]
Can confirm this with recent SDK and master.

**Comment by SaschaWillems** [comment]
Can you recheck if this was fixed with #665?

**Comment by SaschaWillems** [comment]
Closing due to inactivity. If this still persists, feel free to open a new issue.
